ROADMAP.md

```diff
@@ -0,0 +1,104 @@
+# `pflag` roadmap
+
+## Beef up the team of maintainers
+
+3 to 5 volunteers. No less, no more.
+
+Would be nice to open a slack or discord channel to make debate more lively.
+Otherwise, we may use github discussions.
+
+## Agree on the [CONTRIBUTING guidelines](./CONTRIBUTING.md)
+
+There are quite a few debatable things there: let's find a minimal consensus
+(possibly defer decisions about some details to a later stage).
+
+## Agree on a general direction
+
+Proposed principles:
+
+* promise of compatibility (i.e. no breaking `v2` ever)
```

Comment:
I don't think we need to make this promise.

Reply:
@sagikazarmark ok. Still, I'll refrain as much as I can from having to issue a breaking V2

Reply:
That's a good thumb rule, but as soon as you promise it, it's hard to take it back.